app/Console/Commands/MigrateFromSqliteToMysql.php

```
@@ -0,0 +1,191 @@
+            $max = (int) DB::connection($target)->table($table)->max('id');
+            DB::connection($target)->statement("ALTER TABLE `{$table}` AUTO_INCREMENT = ".($max + 1));
+        }
+
+        $this->components->twoColumnDetail("  {$table}", (string) $count.' rows');
+    }
+
+    private function switchEnv(string $target): void
+    {
+        $path = base_path('.env');
+
+        if (! File::exists($path)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\File' in method 'switchEnv'. (undefined)

(StaticAccess)